src/test/regress/specs/isolation_alter_role_propagation.spec

```diff
@@ -0,0 +1,70 @@
+setup
+{
+    CREATE ROLE alter_role_1 WITH LOGIN;
+    SELECT run_command_on_workers($$CREATE ROLE alter_role_1 WITH LOGIN;$$);
+
+    CREATE ROLE alter_role_2 WITH LOGIN;
+    SELECT run_command_on_workers($$CREATE ROLE alter_role_2 WITH LOGIN;$$);
+}
+
+teardown
+{
+    DROP ROLE alter_role_1;
+    SELECT run_command_on_workers($$DROP ROLE alter_role_1;$$);
+
+    DROP ROLE alter_role_2;
+    SELECT run_command_on_workers($$DROP ROLE alter_role_2;$$);
+    
+    SET citus.enable_alter_role_propagation to OFF;
+    SELECT run_command_on_workers($$SET citus.enable_alter_role_propagation to OFF;$$);
+}
+
+session "s1"
+
+step "s1-begin"
+{
+    SET citus.enable_alter_role_propagation to ON;
+    SELECT run_command_on_workers($$SET citus.enable_alter_role_propagation to ON;$$);
```

Comment:
we don't need to set the GUC on the workers. actually, that makes it confusing,  can you remove that?